tool/tctl/common/client/auth.go

```diff
@@ -0,0 +1,127 @@
+/*
+ * Teleport
+ * Copyright (C) 2024  Gravitational, Inc.
+ *
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU Affero General Public License as published by
+ * the Free Software Foundation, either version 3 of the License, or
+ * (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU Affero General Public License for more details.
+ *
+ * You should have received a copy of the GNU Affero General Public License
+ * along with this program.  If not, see <http://www.gnu.org/licenses/>.
+ */
+
+package client
+
+import (
+	"context"
+	"fmt"
+	"os"
+
+	"github.com/gravitational/trace"
+	log "github.com/sirupsen/logrus"
```

Comment:
Does tctl have a slog logger we can use so as not to write new code that depends on logrus?

Reply:
I actually moved part of the code as is, will replace to use `slog` instead